src/__test__/components/ErrorBoundary.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Provider } from 'react-redux';
+import { makeStore } from 'redux/store';
+import postErrorToSlack from 'utils/slack/postErrorToSlack';
+
+import ErrorBoundary from 'components/ErrorBoundary';
+
+const content = 'Some content';
+
+// ErrorBoundary renders the `_error.jsx` component if an error occurs.
+// This is a string contained in the `_error.jsx component to test if the component is rendered.
+const errorText = /Sorry, something went wrong on our end/i;
+
+jest.mock('utils/slack/postErrorToSlack');
+
+delete window.location;
```

Comment:
why is this delete needed? isn't the assignment in next line enough? what is it used for?

Reply:
I think this has to first be deleted because the `location` object under `window` is not [writable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#description). This happens in JSDOM version >= 14, according to this blog post: [https://remarkablemark.org/blog/2018/11/17/mock-window-location/#update-for-jsdom-14](https://remarkablemark.org/blog/2018/11/17/mock-window-location/#update-for-jsdom-14)